app/assets/stylesheets/common/foundation/variables.scss

```diff
@@ -42,37 +42,35 @@ $bronze: #cd7f32 !default;
 // Fonts
 // --------------------------------------------------
 
-$base-font-size-smallest: 0.815em !default; // eq. to 13px
-$base-font-size-smaller: 0.875em !default; // eq. to 14px
-$base-font-size: 0.938em !default; // eq. to 15px
-$base-font-size-larger: 1.063em !default; // eq. to 17px
-$base-font-size-largest: 1.118em !default; // eq. to 19px
 $base-font-family: var(--font-family) !default;
+$base-font-size-smallest: var(--base-font-size-smallest) !default;
+$base-font-size-smaller: var(--base-font-size-smaller) !default;
+$base-font-size: var(--base-font-size) !default;
```

Comment:
I'm assuming the SCSS vars are now legacy, the goal is to start using the CSS vars in stylesheets directly, right?

Reply:
Yep, we can indeed start moving those over :+1: